georiviere/api/valorization/serializers/portal.py

```diff
@@ -0,0 +1,31 @@
+from collections import OrderedDict
+from georiviere.api.valorization.serializers.map import MapBaseLayerSerializer, MapGroupLayerSerializer, MapLayerSerializer
+from georiviere.portal.models import Portal
+
+from rest_framework.serializers import ModelSerializer, SerializerMethodField
+
+
+class PortalSerializer(ModelSerializer):
+    map = MapBaseLayerSerializer(many=True, source='map_base_layers')
+    group = MapGroupLayerSerializer(many=True, source='mapgrouplayer_set')
+    spatial_extent = SerializerMethodField()
+
+    class Meta:
+        model = Portal
+        fields = (
+            'id', 'name', 'map', 'group', 'spatial_extent'
+        )
+
+    def get_spatial_extent(self, obj):
+        return obj.spatial_extent.extent
```

Comment:
exception if no spatial extent defined

Reply:
If there is no spatial extent i give spatial extent of the project 
(settings.Spatial_extent)

Reply:
this is not the case